keps/sig-ui/5008-headlamp/README.md

```diff
@@ -235,11 +240,17 @@ The "Design Details" section below is for the real
 nitty-gritty.
 -->
 
-The proposal is to ship the Headlamp experience as a new Kubernetes Desktop
-and Kubernetes in-cluster UI.
+The proposal is to continue the development of Headlamp as part of the
+Kubernetes project, under the auspices of Kubernetes SIG UI.
+
+This means the Headlamp project repo is moved under the kubernetes-sigs GitHub
+organization.
 
-This means the Headlamp project will become a part of the Kubernetes project
-under the auspices of Kubernetes SIG UI.
+There are two other repositories under the current Headlamp K8s org: _plugins_,
+and _website_; these representing the plugins that maintained by Headlamp's
+core developers, and the website of the project, respectively. The website
+domain is already managed by the CNCF (since Headlamp is a CNCF Sandbox
+project).
 
 ### User Stories
```

Comment:
A bit further in the Story 2 there's a line:
```
Note: We have the minikube installer plugin in POC but the tutorial plugin is not something we currently have.
```
I don't think it matters there, since that section only lists possible user stories, not necessarily mean all will be implemented, iow. drop the note.

Reply:
All the stories are implemented ATM except for that one, hence that note. But I can drop it.

Reply:
I have updated the PR as sugested.